binding_call.go

```diff
@@ -19,7 +19,7 @@ type BindingSource struct {
 type ExposedFunction = func(args ...interface{}) interface{}
 
 // BindingCallFunction represents the func signature of an exposed binding call func
-type BindingCallFunction = func(source *BindingSource, args ...interface{}) interface{}
+type BindingCallFunction func(source *BindingSource, args ...interface{}) interface{}
```

Comment:
whats the reason that this had to be changed? Just curious.

Reply:
The go compiler wasn't able to compile this :/

Reply:
oh interesting. LGTM.